__tests__/integration/atomic-swap.test.js

```diff
@@ -106,9 +110,12 @@ describe('send tx (HTR)', () => {
       .get('/wallet/atomic-swap/tx-proposal/get-locked-utxos')
       .set({ 'x-wallet-id': wallet1.walletId });
     loggers.test.insertLineToLog('atomic-swap[lock]: check with lock', { body: response.body });
+    // XXX: We have an extra utxo which should not be returned since its spent.
     expect(response.body).toEqual({
       success: true,
-      locked_utxos: [],
+      locked_utxos: expect.not.arrayContaining([
```

Comment:
I think these changes are not supposed to happen, we should fix them and keep the old code here.